src/flags.ts

```diff
@@ -0,0 +1,6 @@
+import { flags } from '@oclif/command';
+
+export const watchFlag = flags.boolean({
```

Comment:
it works fine without `Flags.build`? https://oclif.io/docs/flags#custom-flags 🤔

Reply:
@magicmatatjahu , `flags.build`, expects a string value as input to `--watch`, but `--watch` is a boolean.
So I had to go with this

Reply:
Yeah, you're right... They force the string type... 🤦🏼  https://github.com/oclif/core/blob/main/src/parser/flags.ts#L21 Ok, leave it as it is.

Please propagate suggestion from Łukasz (about chokadir and fix text in logs) and ping me for next review :)